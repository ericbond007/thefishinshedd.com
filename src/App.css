.App {
  text-align: center;

}


.App-logo {
  height: 80px;
}

.App-header {
  background-color: #222;
  height: 150px;
  padding: 20px;
  color: white;
}

.App-intro {
  font-size: large;
}


.navbar .navbar-nav {
  display: inline-block;
  float: none;
}

.navbar .navbar-collapse {
  text-align: center;
}

.jumbotron {
  margin-top: 5%;
  margin-bottom: 5%;
}

.pageHeader {
  margin-bottom: 8vh;
}
