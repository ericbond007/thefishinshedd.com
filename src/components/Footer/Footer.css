.footer {
  max-width: 100%;
  background-color: #043025;
  text-align: center;
  min-height: 5vh;
  color: #fff;
}


.footer a {
  color: #fff;
}

.footUl {
  display: inline-block;
  float: none;
  margin: 0;
  padding: 0;
  list-style: none;
}

.footUl li {
  float: left;
  display: block;
  position: relative;
}

.rowBehave {
  margin-top: 1rem;
}



.footUl i {
  padding: unset;
  padding-left: 1rem;
  padding-right: 1rem;
  font-size: 2.6rem;
}

@media only screen and (min-width: 767px) {
}

.version {
  display: none;
  visibility: hidden;
}
