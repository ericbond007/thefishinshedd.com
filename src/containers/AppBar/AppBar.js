import React, { Component } from 'react';
import Header from '../../components/Header/Header';
import NavBar from '../../components/NavBar/NavBar';


class AppBar extends Component {
  render() {
    return (
      <div>
      <Header />
      <NavBar />
      </div>
    );
  }
}

export default AppBar;
