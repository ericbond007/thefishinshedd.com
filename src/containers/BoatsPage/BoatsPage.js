import React, { Component } from 'react';
import Boats from '../../components/Boats/Boats';

class BoatsPage extends Component {
  componentDidMount() {
    window.scrollTo(0, 0);
  }
  render() {
    return (
      <div>
      <Boats />
      </div>
    );
  }
}

export default BoatsPage;
