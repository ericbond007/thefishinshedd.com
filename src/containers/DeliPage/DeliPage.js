import React, { Component } from 'react';
import Deli from '../../components/Deli/Deli';
import DeliBG from '../../components/DeliBG/DeliBG';

class DeliPage extends Component {
  componentDidMount() {
    window.scrollTo(0, 0);
  }
  render() {
    return (
      <div>
      <DeliBG />
      <Deli />
      </div>
    );
  }
}

export default DeliPage;
