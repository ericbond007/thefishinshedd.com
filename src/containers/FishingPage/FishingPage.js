import React, { Component } from 'react';
import Fishing from '../../components/Fishing/Fishing';
import FishingBG from '../../components/FishingBG/FishingBG';

class FishingPage extends Component {
  componentDidMount() {
    window.scrollTo(0, 0);
  }
  render() {
    return (
      <div>
      <FishingBG />
      <Fishing />
      </div>
    );
  }
}

export default FishingPage;
