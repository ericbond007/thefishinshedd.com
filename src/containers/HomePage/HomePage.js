import React, { Component } from 'react';
import Home from '../../components/Home/Home';
import HomeBG from '../../components/HomeBG/HomeBG';

class HomePage extends Component {
  componentDidMount() {
    window.scrollTo(0, 0);
  }
  render() {
    return (
      <div>
        <HomeBG />
        <Home />
      </div>
    );
  }
}

export default HomePage;
