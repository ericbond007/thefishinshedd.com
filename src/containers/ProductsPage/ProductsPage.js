import React, { Component } from 'react';
import Products from '../../components/Products/Products';
import ProductsBG from '../../components/ProductsBG/ProductsBG';

class ProductsPage extends Component {
  componentDidMount() {
    window.scrollTo(0, 0);
  }
  render() {
    return (
      <div>
      <ProductsBG />
      <Products />
      </div>
    );
  }
}

export default ProductsPage;
