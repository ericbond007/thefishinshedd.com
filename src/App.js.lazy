import App from './Shedd';
function errorLoading(err) {
  console.error('Lazy-loading failed', err);
}
function loadRoute(cb) {
  return (module) => cb(null, module.default);
}
export default {
  component: App,
  childRoutes: [
    // ...
    {
      path: 'products',
      getComponent(location, cb) {
        System.import('/components/Products')
          .then(loadRoute(cb))
          .catch(errorLoading);
      }
    },
    {
      path: 'deli',
      getComponent(location, cb) {
        System.import('/components/Deli')
          .then(loadRoute(cb))
          .catch(errorLoading);
      }
    },
    {
      path: 'fishing',
      getComponent(location, cb) {
        System.import('/components/Fishing')
          .then(loadRoute(cb))
          .catch(errorLoading);
      }
    },
    {
      path: 'story',
      getComponent(location, cb) {
        System.import('/components/Story')
          .then(loadRoute(cb))
          .catch(errorLoading);
      }
    },
  ]
};
