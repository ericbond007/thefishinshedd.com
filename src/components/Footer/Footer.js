import React, { Component } from 'react';
import './Footer.css';

class Footer extends Component {
  render() {
    return (
      <div className="footer container-fluid">
        <div className="rowBehave">
          <ul className="footUl">
            <li><p>Call Us! <a href="tel:+1-812-837-9474" aria-label="1-812-837-9474">812-837-9474</a></p></li>
            <li><p><a rel="noopener noreferrer" target="_blank" href="https://www.google.com/maps/place/The+Fishin+Shedd/@39. 0944962,-86.4237921,15z/data=!4m5!3m4!1s0x0:0xc93bcd5938c701d9!8m2!3d39.0926281!4d-86.4224884" >4855 S State Road 446 Bloomingtion, IN 47401</a></p></li>

          <li><p><a href="mailto:info@thefishinshedd.com">Email Us Too! info@thefishinshedd.com</a></p></li>
            <li><a rel="noopener noreferrer" target="_blank" href="https://urlgeni.us/facebook/GTOU"><i className="fa fa-facebook-square" aria-label="https://urlgeni.us/facebook/GTOU"></i></a></li>
            <li><a rel="noopener noreferrer" target="_blank" href="https://urlgeni.us/instagram/DF37"><i className="fa fa-instagram" aria-label="https://urlgeni.us/instagram/DF37"></i> </a></li>
            <li><a rel="noopener noreferrer" target="_blank" href="https://urlgeni.us/snapchat/MzDm"><i className="fa fa-snapchat" aria-label="https://urlgeni.us/snapchat/MzDm"></i> </a></li>
            <li><p>&copy; 2018 The Fishin Shedd</p></li>
          </ul>
      </div>
      <div className="version">
        version 1.0.0
      </div>
      </div>
    );
  }
}

export default Footer;
