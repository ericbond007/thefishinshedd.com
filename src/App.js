import React, { Component } from 'react';
import { BrowserRouter as Router, Route } from 'react-router-dom';

import HomePage from './containers/HomePage/HomePage';
import ProductsPage from './containers/ProductsPage/ProductsPage';
import DeliPage from './containers/DeliPage/DeliPage';
import FishingPage from './containers/FishingPage/FishingPage';
import StoryPage from './containers/StoryPage/StoryPage';
import BoatsPage from './containers/BoatsPage/BoatsPage';
import AppBar from './containers/AppBar/AppBar';
import Footer from './components/Footer/Footer';
import withTracker from './components/withTracker';

import './App.css';


class App extends Component {
  render() {
    return (
      <Router>
      <div>
        <AppBar />
        <Route path="/deli" component={withTracker(DeliPage)} />
        <Route path="/products" component={withTracker(ProductsPage)} />
        <Route path="/story" component={withTracker(StoryPage)} />
        <Route path="/boats" component={withTracker(BoatsPage)} />
        <Route path="/fishing" component={withTracker(FishingPage)} />
        <Route exact path="/" component={withTracker(HomePage)} />


        <Footer />
      </div>
      </Router>
    );
  }
}

export default App;
