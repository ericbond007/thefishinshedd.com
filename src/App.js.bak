import React, { Component } from 'react';
import { BrowserRouter as Router, Route } from 'react-router-dom';

import HomePage from './containers/HomePage/HomePage';
import ProductsPage from './containers/ProductsPage/ProductsPage';
import DeliPage from './containers/DeliPage/DeliPage';
import FishingPage from './containers/FishingPage/FishingPage';
import StoryPage from './containers/StoryPage/StoryPage';
import AppBar from './containers/AppBar/AppBar';
import Footer from './components/Footer/Footer';
import withTracker from './components/withTracker';

import './App.css';


class App extends Component {
  render() {
    return (
      <Router>
      <div>
        <AppBar />
        <Route path="/deli" component={withTracker(DeliPage)} />
        <Route path="/products" component={withTracker(ProductsPage)} />
        <Route path="/story" component={withTracker(StoryPage)} />
        <Route path="/fishing" component={withTracker(FishingPage)} />


        <Route component={Footer} />
      </div>
      </Router>
    );
  }
}

export default App;
