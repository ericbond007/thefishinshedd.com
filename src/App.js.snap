import React, { Component } from 'react';
import { BrowserRouter as Router, Route } from 'react-router-dom';

import HomePage from './containers/HomePage/HomePage';
import ProductsPage from './containers/ProductsPage/ProductsPage';
import DeliPage from './containers/DeliPage/DeliPage';
import FishingPage from './containers/FishingPage/FishingPage';
import StoryPage from './containers/StoryPage/StoryPage';
import AppBar from './containers/AppBar/AppBar';
import Footer from './components/Footer/Footer';
import withTracker from './components/withTracker';

import './App.css';


class App extends Component {
  render() {
    return (
      <Router>
      <div>
        <Route path="/" component={HomePage} />


      </div>
      </Router>
    );
  }
}

export default App;
